### ISSUE TITLE
WCAG 2.1 Compliance

### ISSUE THREAD

**Issue by gjanblaszczyk** [videojs/video.js#5732]
## Description
This is a question. I am not a WCAG expert.
May I know whether the video.js player is WCAG 2.1 compliant?
I know that the player should be WCAG 2.0 compliant but is it ready for WCAG 2.1?

**Comment by gkatsev** [comment]
I think we are likely compliant or striving to be compliant. Though, @OwenEdwards may have more thoughts on the subject.

**Comment by OwenEdwards** [comment]
We haven't focused on WCAG 2.1 compliance yet. Feel free to report any issues you see which might violate WCAG 2.1 Level A or AA.

**Comment by gkatsev** [comment]
Going to close this in favor of #5947. We're committed to always improving the accessibility of the player.
